src/ru/skypro/Main.java

```diff
@@ -4,5 +4,60 @@
 
     public static void main(String[] args) {
 	// write your code here
+        System.out.println("Задача 1");
+        byte child = 1;
+        short men = 200;
+        int women = 500;
+        long people = 1000L;
+        float height = 2.55f;
+        double weight = 50.7777777;
+        System.out.println("Задача 2");
+        float firstBoxer = 78.2f;
+        float secondBoxer = 82.7f;
+        float totalWeight = firstBoxer + secondBoxer;
+        System.out.println("Общий вес " + totalWeight + " кг");
+        float raznica = secondBoxer - firstBoxer;
+        System.out.println("Разница в весе " + raznica + " кг");
+        System.out.println("Задача 3");
+        int banan = 80;
+        int milk = 105;
+        int ice = 100;
+        int egg = 70;
+        int total = banan * 5 + milk * 2 + ice * 2 + egg * 4;
+        float kilo = total / 1000;
```

Comment:
хоть результат получается и целым числом, но дробное также могло иметь место. В этом случае нужно привести один аз аргументов к float, иначе результат `int / int = int`
```suggestion
        float kilo = (float) total / 1000;
```